React/CoreModules/RCTDeviceInfo.mm

```diff
@@ -163,9 +172,9 @@ - (void)didReceiveNewContentSizeMultiplier
 #pragma clang diagnostic ignored "-Wdeprecated-declarations"
     [bridge.eventDispatcher sendDeviceEventWithName:@"didUpdateDimensions"
 #if !TARGET_OS_OSX // TODO(macOS GH#774)
-    body:RCTExportedDimensions(bridge)];
+                                               body:RCTExportedDimensions(bridge, self->_turboModuleRegistry)];
```

Comment:
This is the desired spacing here and below?

Reply:
Yeah, this is intentional. Upstream, these are spaced out so the colons line up with each other.